backend/experiment/actions/playback.py

```diff
@@ -12,16 +12,18 @@ class Playback(BaseAction):
         - instruction: text to display during presentation of the sound
         - play_config: define to override the following values:
             - play_method:
-                - 'BUFFER': Use webaudio buffers. (recommended for stimuli up to 45s)  
+                - 'BUFFER': Use webaudio buffers. (recommended for stimuli up to 45s)
                 - 'HTML': Use the HTML tag. (recommended for stimuli longer than 45s)
-                - 'EXTERNAL': Use for externally hosted audio files. Web-audio api will be disabled            
+                - 'EXTERNAL': Use for externally hosted audio files. Web-audio api will be disabled
+                - 'MATCHINGPAIRS': Use for the matching pairs game.
```

Comment:
I believe the `MATCHINGPAIRS` play_method isn't used anymore. In https://github.com/Amsterdam-Music-Lab/MUSCLE/pull/648, the play_method is determined based on the type of resource (i.e., `EXTERNAL` has "http" in the filename, `BUFFER` are files of duration up to 45s, `HTML` is everything longer than that (since web audio cannot buffer longer files)) - we could perhaps make the `PREFETCH` method depend on whether the extension is an image file? Just noting this down for future reference. My bad for not having been able to merge the redesigned play config yet. NB: it won't work to use the admin interface to add images in the current setup, as that presumes upload of audio files. Not a problem per se, but something to think about for the future.

Reply:
Ah, I think I should move it to the `player_type` docs then, or not?